Assets/Scenes/Level1.unity

```diff
@@ -26,7 +26,7 @@ RenderSettings:
   m_AmbientIntensity: 1
```

Comment:
Make sure to add the skybox to Level 2 and 3 too

Reply:
done